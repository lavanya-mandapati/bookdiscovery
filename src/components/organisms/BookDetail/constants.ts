export interface Desc {
  [name: string]: string;
}

export const description: Desc = {
  desc1:
    "Through a series of recent breakthroughs, deep learning has boosted the entire field of machine learning.Now, even programmers who know close to nothing about this technology can use simple, efficient tools to implement programs capable of learning from data. This practical book shows you how.",
  desc2:
    "By using concrete examples, minimal theory, and two production-ready Python frameworks—Scikit-Learn and TensorFlow—author Aurélien Géron helps you gain an intuitive understanding of the concepts and tools for building intelligent systems. You’ll learn a range of techniques, starting with simple linear regression and progressing to deep neural",
};
