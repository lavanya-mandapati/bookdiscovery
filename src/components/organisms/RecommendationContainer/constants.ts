import { RecommendationBookCardProps } from "../RecommendationBookCard/RecommendationBookCard";

export const bookData: RecommendationBookCardProps[] = [
    {
      title: "Crystal chemistry",
      authorName: "Joe Pearson",
      image:
        "https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/E3EEB4FA-4DF1-491D-8A51-DC1C1921FB60.png",
    },
    {
      title: "Inorganic chemistry",
      authorName: "Joe Pearson",
      image:
        "https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/03991C69-3988-432F-AD72-535D7FCFFC54.png",
    },
    {
      title: "Bioorganic chemistry",
      authorName: "Joe Pearson",
      image:
        "https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/19118712-5417-4364-9FC6-59FA289CE148.png",
    },
    {
      title: "Basic Physics",
      authorName: "Joe Pearson",
      image:
        "https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/BFB91888-5030-4E35-878F-68EC357D95CE.png",
    },
  ];