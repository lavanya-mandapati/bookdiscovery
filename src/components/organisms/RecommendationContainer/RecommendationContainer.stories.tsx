import React from "react";
import {bookData} from './constants';
import RecommendationContainer from "./RecommendationContainer";

export default {
  title: "organisms/RecommendationContainer",
  component: RecommendationContainer,
};

const Template = () => <RecommendationContainer data={bookData} />;

export const Default = Template.bind({});
