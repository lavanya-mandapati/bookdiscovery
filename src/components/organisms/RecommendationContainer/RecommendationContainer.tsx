import React from "react";
import RecommendationBookCard, { RecommendationBookCardProps } from "../RecommendationBookCard/RecommendationBookCard";
import { Grid, makeStyles, Typography } from "@material-ui/core";
import { COLORS } from '../../../theme/constants';
import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
import theme from "../../../theme/theme";

export type RecommendationContainerProps={
    cardClicked?: ()=>any;
    data: Array<RecommendationBookCardProps>;
}
const useStyles = makeStyles({
    heading: {
        width: "475px",
        height: "38px",
        margin: theme.spacing(4,3,1,20),
        color: COLORS.CHARCOAL_GREY_100
    },
    text: {
        width: "68px",
        height: "24px",
        margin: theme.spacing(5,2,1,125),
        color: COLORS.PRIMARY_COLOR_500,
    },
    icon: {
        color: COLORS.PRIMARY_COLOR_500,
        margin: theme.spacing(7,2,1,0),
        fontSize: "12px"
    },
    textContainer:{
        display: "flex"
    },
    grid:{
        display: "flex",
        marginLeft: theme.spacing(16),
        marginRight: theme.spacing(20)
    }
})
const RecommendationContainer: React.FC<RecommendationContainerProps> = ({
    data
}) => {
    const classes = useStyles();
    const recommendationCards = data.map((dataItem) => {
        return <RecommendationBookCard
            authorName={dataItem.authorName}
            title={dataItem.title}
            image={dataItem.image} />
    })
    return (
        <React.Fragment>
            <Grid className={classes.textContainer}>
            <Typography variant="h3" className={classes.heading}>Recommendations based on your search</Typography>
            <Typography variant="subtitle2" className={classes.text}>See more</Typography>
            <ArrowForwardIosIcon className={classes.icon} />
            </Grid>
            <Grid className={classes.grid}>
                   {recommendationCards}
            </Grid>
        </React.Fragment>
    );
}

export default RecommendationContainer;