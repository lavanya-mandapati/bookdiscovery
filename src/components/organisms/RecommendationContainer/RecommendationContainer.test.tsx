import { render } from "@testing-library/react";
import { bookData } from "./constants";
import RecommendationContainer from "./RecommendationContainer";



describe("Should render container correctly", () => {
  const component = <RecommendationContainer data={bookData} />;

  it("Match Snapshot", () => {
    const { container } = render(<RecommendationContainer data={bookData} />);
    expect(container).toMatchSnapshot();
  });

  test("Should render correctly", () => {
    const { container } = render(<RecommendationContainer data={bookData} />);
    expect(container).toBeInTheDocument();
  });
});
