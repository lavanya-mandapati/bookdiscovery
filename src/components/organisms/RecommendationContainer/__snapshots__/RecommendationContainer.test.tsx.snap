// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render container correctly Match Snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root makeStyles-textContainer-4"
  >
    <h3
      class="MuiTypography-root makeStyles-heading-1 MuiTypography-h3"
    >
      Recommendations based on your search
    </h3>
    <h6
      class="MuiTypography-root makeStyles-text-2 MuiTypography-subtitle2"
    >
      See more
    </h6>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root makeStyles-icon-3"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M5.88 4.12L13.76 12l-7.88 7.88L8 22l10-10L8 2z"
      />
    </svg>
  </div>
  <div
    class="MuiGrid-root makeStyles-grid-5"
  >
    <div
      class="MuiPaper-root MuiCard-root makeStyles-root-6 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <td
        class="MuiTableCell-root makeStyles-cell-20"
      >
        <img
          class="MuiCardMedia-root makeStyles-image-7 MuiCardMedia-media MuiCardMedia-img"
          src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/E3EEB4FA-4DF1-491D-8A51-DC1C1921FB60.png"
        />
      </td>
      <div
        class="MuiCardContent-root makeStyles-content-19"
      >
        <div
          class="MuiCardActions-root makeStyles-titleActions-14 MuiCardActions-spacing"
        >
          <h6
            class="MuiTypography-root makeStyles-title-8 MuiTypography-h6"
          >
            Crystal chemistry
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <p
            class="MuiTypography-root makeStyles-author-9 MuiTypography-body2"
          >
            Author
          </p>
          <h6
            class="MuiTypography-root makeStyles-authorName-10 MuiTypography-subtitle2"
          >
            Joe Pearson
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <span
            aria-label="4 Stars"
            class="MuiRating-root MuiRating-sizeSmall MuiRating-readOnly"
            role="img"
          >
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconEmpty"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
          </span>
          <p
            class="MuiTypography-root makeStyles-star-11 MuiTypography-body1"
          >
            4.0
          </p>
          <span
            class="MuiTypography-root makeStyles-rating-12 MuiTypography-overline"
          >
            (1,204 ratings)
          </span>
        </div>
      </div>
      <hr
        class="MuiDivider-root"
      />
      <div
        class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-bookmarkBorder-16"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M17 3H7c-1.1 0-1.99.9-1.99 2L5 21l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-reply-17"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 9V5l-7 7 7 7v-4.1c5 0 8.5 1.6 11 5.1-1-5-4-10-11-11z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-playlist-18"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M13 10H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm0-4H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm5 8v-3c0-.55-.45-1-1-1s-1 .45-1 1v3h-3c-.55 0-1 .45-1 1s.45 1 1 1h3v3c0 .55.45 1 1 1s1-.45 1-1v-3h3c.55 0 1-.45 1-1s-.45-1-1-1h-3zM3 16h6c.55 0 1-.45 1-1s-.45-1-1-1H3c-.55 0-1 .45-1 1s.45 1 1 1z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiCard-root makeStyles-root-6 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <td
        class="MuiTableCell-root makeStyles-cell-20"
      >
        <img
          class="MuiCardMedia-root makeStyles-image-7 MuiCardMedia-media MuiCardMedia-img"
          src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/03991C69-3988-432F-AD72-535D7FCFFC54.png"
        />
      </td>
      <div
        class="MuiCardContent-root makeStyles-content-19"
      >
        <div
          class="MuiCardActions-root makeStyles-titleActions-14 MuiCardActions-spacing"
        >
          <h6
            class="MuiTypography-root makeStyles-title-8 MuiTypography-h6"
          >
            Inorganic chemistry
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <p
            class="MuiTypography-root makeStyles-author-9 MuiTypography-body2"
          >
            Author
          </p>
          <h6
            class="MuiTypography-root makeStyles-authorName-10 MuiTypography-subtitle2"
          >
            Joe Pearson
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <span
            aria-label="4 Stars"
            class="MuiRating-root MuiRating-sizeSmall MuiRating-readOnly"
            role="img"
          >
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconEmpty"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
          </span>
          <p
            class="MuiTypography-root makeStyles-star-11 MuiTypography-body1"
          >
            4.0
          </p>
          <span
            class="MuiTypography-root makeStyles-rating-12 MuiTypography-overline"
          >
            (1,204 ratings)
          </span>
        </div>
      </div>
      <hr
        class="MuiDivider-root"
      />
      <div
        class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-bookmarkBorder-16"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M17 3H7c-1.1 0-1.99.9-1.99 2L5 21l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-reply-17"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 9V5l-7 7 7 7v-4.1c5 0 8.5 1.6 11 5.1-1-5-4-10-11-11z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-playlist-18"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M13 10H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm0-4H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm5 8v-3c0-.55-.45-1-1-1s-1 .45-1 1v3h-3c-.55 0-1 .45-1 1s.45 1 1 1h3v3c0 .55.45 1 1 1s1-.45 1-1v-3h3c.55 0 1-.45 1-1s-.45-1-1-1h-3zM3 16h6c.55 0 1-.45 1-1s-.45-1-1-1H3c-.55 0-1 .45-1 1s.45 1 1 1z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiCard-root makeStyles-root-6 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <td
        class="MuiTableCell-root makeStyles-cell-20"
      >
        <img
          class="MuiCardMedia-root makeStyles-image-7 MuiCardMedia-media MuiCardMedia-img"
          src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/19118712-5417-4364-9FC6-59FA289CE148.png"
        />
      </td>
      <div
        class="MuiCardContent-root makeStyles-content-19"
      >
        <div
          class="MuiCardActions-root makeStyles-titleActions-14 MuiCardActions-spacing"
        >
          <h6
            class="MuiTypography-root makeStyles-title-8 MuiTypography-h6"
          >
            Bioorganic chemistry
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <p
            class="MuiTypography-root makeStyles-author-9 MuiTypography-body2"
          >
            Author
          </p>
          <h6
            class="MuiTypography-root makeStyles-authorName-10 MuiTypography-subtitle2"
          >
            Joe Pearson
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <span
            aria-label="4 Stars"
            class="MuiRating-root MuiRating-sizeSmall MuiRating-readOnly"
            role="img"
          >
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconEmpty"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
          </span>
          <p
            class="MuiTypography-root makeStyles-star-11 MuiTypography-body1"
          >
            4.0
          </p>
          <span
            class="MuiTypography-root makeStyles-rating-12 MuiTypography-overline"
          >
            (1,204 ratings)
          </span>
        </div>
      </div>
      <hr
        class="MuiDivider-root"
      />
      <div
        class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-bookmarkBorder-16"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M17 3H7c-1.1 0-1.99.9-1.99 2L5 21l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-reply-17"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 9V5l-7 7 7 7v-4.1c5 0 8.5 1.6 11 5.1-1-5-4-10-11-11z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-playlist-18"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M13 10H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm0-4H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm5 8v-3c0-.55-.45-1-1-1s-1 .45-1 1v3h-3c-.55 0-1 .45-1 1s.45 1 1 1h3v3c0 .55.45 1 1 1s1-.45 1-1v-3h3c.55 0 1-.45 1-1s-.45-1-1-1h-3zM3 16h6c.55 0 1-.45 1-1s-.45-1-1-1H3c-.55 0-1 .45-1 1s.45 1 1 1z"
          />
        </svg>
      </div>
    </div>
    <div
      class="MuiPaper-root MuiCard-root makeStyles-root-6 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <td
        class="MuiTableCell-root makeStyles-cell-20"
      >
        <img
          class="MuiCardMedia-root makeStyles-image-7 MuiCardMedia-media MuiCardMedia-img"
          src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/BFB91888-5030-4E35-878F-68EC357D95CE.png"
        />
      </td>
      <div
        class="MuiCardContent-root makeStyles-content-19"
      >
        <div
          class="MuiCardActions-root makeStyles-titleActions-14 MuiCardActions-spacing"
        >
          <h6
            class="MuiTypography-root makeStyles-title-8 MuiTypography-h6"
          >
            Basic Physics
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <p
            class="MuiTypography-root makeStyles-author-9 MuiTypography-body2"
          >
            Author
          </p>
          <h6
            class="MuiTypography-root makeStyles-authorName-10 MuiTypography-subtitle2"
          >
            Joe Pearson
          </h6>
        </div>
        <div
          class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
        >
          <span
            aria-label="4 Stars"
            class="MuiRating-root MuiRating-sizeSmall MuiRating-readOnly"
            role="img"
          >
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconFilled"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
            <span>
              <span
                class="MuiRating-icon MuiRating-iconEmpty"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                  />
                </svg>
              </span>
            </span>
          </span>
          <p
            class="MuiTypography-root makeStyles-star-11 MuiTypography-body1"
          >
            4.0
          </p>
          <span
            class="MuiTypography-root makeStyles-rating-12 MuiTypography-overline"
          >
            (1,204 ratings)
          </span>
        </div>
      </div>
      <hr
        class="MuiDivider-root"
      />
      <div
        class="MuiCardActions-root makeStyles-actions-13 MuiCardActions-spacing"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-bookmarkBorder-16"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M17 3H7c-1.1 0-1.99.9-1.99 2L5 21l7-3 7 3V5c0-1.1-.9-2-2-2zm0 15l-5-2.18L7 18V5h10v13z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-reply-17"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 9V5l-7 7 7 7v-4.1c5 0 8.5 1.6 11 5.1-1-5-4-10-11-11z"
          />
        </svg>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-playlist-18"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M13 10H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm0-4H3c-.55 0-1 .45-1 1s.45 1 1 1h10c.55 0 1-.45 1-1s-.45-1-1-1zm5 8v-3c0-.55-.45-1-1-1s-1 .45-1 1v3h-3c-.55 0-1 .45-1 1s.45 1 1 1h3v3c0 .55.45 1 1 1s1-.45 1-1v-3h3c.55 0 1-.45 1-1s-.45-1-1-1h-3zM3 16h6c.55 0 1-.45 1-1s-.45-1-1-1H3c-.55 0-1 .45-1 1s.45 1 1 1z"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
